Schema/labels.schema.json

```diff
@@ -14,22 +14,18 @@
         },
         "url": {
             "type": "string",
-            "format": "uri"
+            "format": "url"
```

Comment:
Hey, I don't quite get why you changed the format?

The github api says format: uri. 

The difference between a url and uri is a url is basically just a address, while an uri had more usecases